### ISSUE TITLE
BUG: Couldn't build processors to update external chain info in the Pigeon Logs (catch-all)

### ISSUE THREAD

**Issue by taariq** [palomachain/paloma#579]
# What is happening?
<details>
  <summary>Section description</summary>
<i>Provide as much context as you can. Give as much context as you can to make it easier for the developers to figure what is happening.</i>
</details>

Pigeons logs are returning this bug text:
time="2022-12-05T15:41:59Z" level=info msg="relayer loop"
time="2022-12-05T15:42:00Z" level=error msg="error while trying to process messages" error="not found"
time="2022-12-05T15:42:00Z" level=error msg="couldn't build processors to update external chain info" err="not found"
time="2022-12-05T15:42:59Z" level=info msg="relayer loop"
time="2022-12-05T15:43:00Z" level=error msg="couldn't build processors to update external chain info" err="not found"
time="2022-12-05T15:43:00Z" level=error msg="error while trying to process messages" error="not found"

# Paloma and pigeon versions and logs
<details>
  <summary>Section description</summary>
<i>Write down paloma version.</i>
<i>Write down pigeon version.</i>
<i>Copy and paste pigeon config file as well as relevant ENV variables.</i>
</details>

Paloma: 0.11.4
Pigeon: 0.12.0

# How to reproduce?
<details>
  <summary>Section description</summary>
  <i>Please write detailed steps of what you were doing for this bug to appear.</i>
</details>

Upgrade and check pigeon logs. Please add your logs in the comment.

# What is the expected behaviour?
<details>
  <summary>Section description</summary>
<i>If you know, please write down what is the expected behaviour. If you don't know, that's ok. We can have a discussion in comments.</i>
</details>

Pigeon must see the EVM chain and be able to create relay messages.

**Comment by DasRasyo** [comment]
paloma version: v0.11.4
pigeon version: v0.12.0

```time="2022-12-05T18:28:47Z" level=info msg="relaying messages" messages-to-relay="[6706]" queue-name=evm/bnb-main/evm-turnstone-message
time="2022-12-05T18:28:47Z" level=error msg="error filtering logs" error="You cannot query logs for more than 100000 blocks at once."
time="2022-12-05T18:28:47Z" level=error msg="processing error" chain-reference-id=bnb-main error="original error: there are 2 errors in the group:\nerror  1: EVM related error: could not find the valset_id in EVM logs\nerror  2: You cannot query logs for more than 100000 blocks at once.\n\nenriched fields:\nmessage id[uint64] = 0x1a32\nmessage type[*types.Message_UpdateValset] = &types.Message_UpdateValset{UpdateValset:(*types.UpdateValset)(0xc0034f3570)}\n" message-type="*types.Message_UpdateValset" msg-id=6706 queue-name=evm/bnb-main/evm-turnstone-message
time="2022-12-05T18:28:47Z" level=error msg="error relaying messages" err="there are 1 errors in the group:\nerror  1: original error: there are 2 errors in the group:\nerror  1: EVM related error: could not find the valset_id in EVM logs\nerror  2: You cannot query logs for more than 100000 blocks at once.\n\nenriched fields:\nmessage id[uint64] = 0x1a32\nmessage type[*types.Message_UpdateValset] = &types.Message_UpdateValset{UpdateValset:(*types.UpdateValset)(0xc0034f3570)}\n\n" messages-to-relay="[6706]" queue-name=evm/bnb-main/evm-turnstone-message
time="2022-12-05T18:28:47Z" level=error msg="error returned in process loop" err="there are 1 errors in the group:\nerror  1: original error: there are 2 errors in the group:\nerror  1: EVM related error: could not find the valset_id in EVM logs\nerror  2: You cannot query logs for more than 100000 blocks at once.\n\nenriched fields:\nmessage id[uint64] = 0x1a32\nmessage type[*types.Message_UpdateValset] = &types.Message_UpdateValset{UpdateValset:(*types.UpdateValset)(0xc0034f3570)}\n\n"
time="2022-12-05T18:29:44Z" level=info msg="relayer loop"
time="2022-12-05T18:29:45Z" level=error msg="error while trying to process messages" error="not found"
time="2022-12-05T18:30:44Z" level=info msg="relayer loop"
time="2022-12-05T18:31:44Z" level=info msg="relayer loop"
time="2022-12-05T18:31:46Z" level=error msg="error while trying to process messages" error="not found"
time="2022-12-05T18:32:44Z" level=info msg="relayer loop"
time="2022-12-05T18:32:46Z" level=error msg="error while trying to process messages" error="not found"
time="2022-12-05T18:33:44Z" level=info msg="relayer loop"
time="2022-12-05T18:33:46Z" level=error msg="error while trying to process messages" error="not found"
time="2022-12-05T18:34:44Z" level=info msg="relayer loop"
time="2022-12-05T18:34:46Z" level=error msg="error while trying to process messages" error="not found"
time="2022-12-05T18:35:44Z" level=info msg="relayer loop"
time="2022-12-05T18:35:45Z" level=error msg="error while trying to process messages" error="not found"```

**Comment by taariq** [comment]
Fixed with #610

**Comment by taariq** [comment]
Closing as fixed with #611 #610 #613 and others.
